integration-test/src/main/scala/com/cloudera/livy/test/framework/BaseIntegrationTestSuite.scala

```diff
@@ -97,17 +97,18 @@ abstract class BaseIntegrationTestSuite extends FunSuite with Matchers {
     }
   }
 
-  test("initialize test cluster") {
+  override def beforeAll() {
```

Comment:
Do you really need this change? I changed this because, if the cluster fails to start, `beforeAll` makes is really, really hard to debug it, since the exception happens during discovery and scalatest doesn't generated good logs in that case.